### PR TITLE
Reference branch for Qt 5.15.(1|2), switch default default to 5.15.2

### PR DESCRIPTION
**Pull request by jcfr**

### CODE REVIEW

**Comment by jcfr** [comment]
cc: @sjh26  @jamesobutler

**Comment by jamesobutler** [comment]
We should revert https://github.com/jcfr/qt-easy-build/commit/d9ae8b83ac08dc8dce4d655a98323f43db431a97 on the Qt 5.15.2 branch since the Linux azure pipelines build failed because of that commit. As related to https://github.com/jcfr/qt-easy-build/pull/60 we still don't have a working solution in the build options.

File: README.md [hunk 1]
```diff
@@ -7,30 +7,32 @@ You will find here scripts allowing to very easily build Qt with OpenSSL support
...
 |--------------|---------------------------------------------------------|---------------------------------------------------------|------------|
-| 5.15.1       | [![Build Status][5151_linux_i_azure]][5151_linux_azure] | [![Build Status][5151_macos_i_azure]][5151_macos_azure] | [![Build Status][5151_windows_i_azure]][5151_windows_azure]         |
+| 5.15.2       | [![Build Status][5152_linux_i_azure]][5152_linux_azure] | [![Build Status][5152_macos_i_azure]][5152_macos_azure] | [![Build Status][5152_windows_i_azure]][5152_windows_azure]         |
 | 4.8.7        | NA                                                      | NA                                                      | NA         |
 
-[5151_linux_azure]: https://dev.azure.com/jamesobutler/qt-easy-build/_build/latest?definitionId=1&branchName=5.15.1
-[5151_linux_i_azure]: https://dev.azure.com/jamesobutler/qt-easy-build/_apis/build/status/jamesobutler.qt-easy-build?branchName=5.15.1&jobName=Linux
+[5152_linux_azure]: https://dev.azure.com/jamesobutler/qt-easy-build/_build/latest?definitionId=1&branchName=5.15.2
```
Comment (jamesobutler) [inline README.md:21]: This should point to your azure qt-easy-build repo. For some reason the last build information can't be obtained. Did you manually delete it? https://dev.azure.com/jchrisfillionr/qt-easy-build/_build/results?buildId=2&jobId=019c090b-adc9-5d3d-bff9-f0ed5003c51c

Comment (jcfr) [reply README.md:21]: >Did you manually delete it?

I don't think so. The only build there is this one:

image

Comment (jamesobutler) [reply README.md:21]: Ok I was able to get a build running https://dev.azure.com/jchrisfillionr/qt-easy-build/_build/results?buildId=3&view=results as part of #64.

**Pull request merged**